src/datachain/lib/dc.py

```diff
@@ -348,6 +348,9 @@ def settings(
                 enable all available CPUs (default=1)
             workers : number of distributed workers. Only for Studio mode. (default=1)
             min_task_size : minimum number of tasks (default=1)
+            prefetch: number of workers to use for downloading files in advance.
```

Comment:
workers in this context might sound like actual DC workers (especially in Studio context). Worker in this case is probably an internal term (and also quite nuanced since we use async for this?). Is there a better way to phrase this?

can we just say: 

`number of files to download in advance for `map`, `gen` operations when `File` object is passed to them`

Reply:
Also, probably better to mention about this option existence in the map / gen also.